src/components/shared/DialogSelectedNote.tsx

```
@@ -155,13 +157,20 @@ const DialogSelectedNote = ({
               {content ? (
-                <div className="prose dark:prose-invert max-w-none">
-                  {content.split('\n').map((paragraph, i) => (
-                    <p key={i} className="mb-4 last:mb-0">
-                      {paragraph || <br />}
-                    </p>
-                  ))}
-                </div>
+                note.isHTML ? (
+                  <div
+                    className="prose prose-lg dark:prose-invert max-w-none"
+                    dangerouslySetInnerHTML={{ __html: content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)